cmd/snap-bootstrap/main.go

```diff
@@ -40,6 +41,11 @@ such as initramfs.
 )
 
 func main() {
+	if err := secboot.MaybeRunArgon2OutOfProcessRequestHandler(); err != nil {
+		fmt.Fprintf(os.Stderr, "cannot run argon2 out-of-process helper command: %v", err)
```

Comment:
it's a bit unclear to me why we don't want to deal with this inside the helper directly

Reply:
Thanks, updated!